libavutil/pixfmt.h

```diff
@@ -348,6 +348,9 @@ enum AVPixelFormat {
     AV_PIX_FMT_NV24,      ///< planar YUV 4:4:4, 24bpp, 1 plane for Y and 1 plane for the UV components, which are interleaved (first byte U and the following byte V)
     AV_PIX_FMT_NV42,      ///< as above, but U and V bytes are swapped
 
+    AV_PIX_FMT_A2R10G10B10, ///< 10-bit Pixel RGB formats.
+    AV_PIX_FMT_A2B10G10R10, ///< 10-bit Pixel BGR formats.
```

Comment:
av_pix_fmt_descriptors should be added for new pixel formats.

And it will break fate if you don't get it support in sws-pixdesc-query.

Reply:
Thank you Linjie for the kind reminder, Yes, av_pix_fmt_descriptors should be added when you try to dump or read in raw frames as my understand. will change this patch or send another patch for av_pix_fmt_descriptors. :)

Reply:
xinpeng's patch will include this: https://github.com/intel-media-ci/ffmpeg/pull/104